### PR TITLE
[PCGenFrame] Use JavaFx browser for "simple" HTML content.

### PR DESCRIPTION
**Pull request by grimreaper**

### CODE REVIEW

File: code/src/java/pcgen/gui3/SimpleHtmlPanel.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package pcgen.gui3;
```
Comment (thpr) [inline code/src/java/pcgen/gui3/SimpleHtmlPanel.java]: Needs a copyright header

File: build.gradle [hunk 2]
```diff
@@ -180,6 +186,7 @@ dependencies {
 
     compileOnly group: 'org.jetbrains', name: 'annotations', version:'17.0.0'
     compileOnly group: 'com.yuvimasory', name: 'orange-extensions', version: '1.3.0'
+    compileOnly group: 'com.github.spotbugs', name: 'spotbugs-annotations', version: '3.1.12'
```
Comment (thpr) [inline build.gradle]: This seems out of place in a browser swap...

Comment (grimreaper) [reply build.gradle]: I was going for "all new gui3 code will use the new rules..." but yeah, this should be a different, followup, change.

**Comment by grimreaper** [comment]
> Also, can we drop any of the libraries with this change?

With the changes in #5408, yes.

**Pull request merged**